mnefun/_mnefun.py

```diff
@@ -854,6 +898,83 @@ def run_sss_positions(fname_in, fname_out, host='kasga', opts='', port=22):
     run_subprocess(cmd)
 
 
+def run_sss_localy(p, subjects, run_indices):
```

Comment:
this one is okay to rely on `params`, though, because it's only meant to be used with the analysis scripts